docs/meetings.md

```diff
@@ -0,0 +1,56 @@
+# Meetings
+
+Our team meetings are organised similarly each week. We have [Standup](#standup) meetings on Monday, Tuesday and Thursday morning and all the other team meetings happen on Thursdays to make our team schedule compact.
+
+:::tip
+As part of our role-rotation practice, all the regular weekly meetings are facilities by a Kanban lead](./roles#kanban-lead) of the week.
+:::
```

Comment:
```suggestion
:::tip

As part of our role-rotation practice, all the regular weekly meetings are facilitated by the Kanban Lead](./roles#kanban-lead) of the week.

:::
```

New lines because of potential formatting and breaking the syntax.